app/views/content/non-uk-teachers/teach-in-england-if-you-trained-overseas.md

```diff
@@ -63,7 +63,7 @@ You’ll earn a [competitive salary](/salaries-and-benefits), get supported in y
 
 You could also consider teaching in [Wales](https://educators.wales), [Scotland](https://teachinscotland.scot/) or [Northern Ireland](https://www.education-ni.gov.uk/articles/initial-teacher-education-courses-northern-ireland).
 
-Not qualified as a teacher yet? Find out [how to train to teach in England as an international applicant](/non-uk-teachers/train-to-teach-in-england-as-an-international-student). If you don't want to come to England to train, but would like to gain a professional teaching qualification which is equivalent to English qualified teacher status, you should consider [international qualified teacher status (iQTS)](/non-uk-teachers/international-qualified-teacher-status).
+Not qualified as a teacher yet? Find out [how to train to teach in England as a non-UK applicant or international student](/non-uk-teachers/train-to-teach-in-england-as-an-international-student). If you don't want to come to England to train, but would like to gain a professional teaching qualification which leads to English qualified teacher status, you should consider [international qualified teacher status (iQTS)](/non-uk-teachers/international-qualified-teacher-status).
```

Comment:
@laurahermionetennant92 what does 'non-UK applicant' actually mean? Does it mean they don't live in the UK or they're not from the UK? What if a non-UK citizen did their undergrad at a UK university and is looking to train to teach in England - are they classed as a non-UK applicant or not? (sorry if I have confused things further!!)

Reply:
No fair point lols. How about
Not qualified as a teacher yet? Find out how to [apply to train to teach in England] as:
* an international student living in the UK
* a non-UK citizen living in the UK
* a non-UK citizen applying from overseas
If you don't want to... etc

I can't find Steph's UR (well I can find it but I can't open it for some reason) but from memory the research she did found that non-UK citizens living in the UK who want to change career etc did not recognise themselves in the various labels we currently use...

 


 'International student' refers to 'foreign students in the UK' and 'non-UK applicant' to 'non-UK citizen living in the UK' but may

Reply:
hmm wanna send me a link and I'll see if I can open it/message Steph to see what's happening?

Reply:
@laurahermionetennant92 could we maybe just use 'non-UK citizens' as a catch all term? I _think_ this would then apply to all of the user groups you've listed above? might be a lil simpler just to have one term (if we're just referring to people who come from outside the UK in general, regardless of where they live/study now)

Reply:
If that works from an SEO perspective then yes I think that's a great idea :)